src/abstract/ThresholdGovernor.sol

```diff
@@ -61,6 +62,11 @@ abstract contract ThresholdGovernor is IThresholdGovernor, BatchGovernor {
 
     /* ============ View/Pure Functions ============ */
 
+    /// @inheritdoc IGovernor
+    function COUNTING_MODE() external pure returns (string memory) {
+        return "support=against,for&quorum=for&success=quorum";
```

Comment:
how about `success=threshold` ?

Reply:
While it could work, then we would need to expose `proposalThreshold(proposalID)`, which is already reserved for the amount of tokens you need to create a proposal. For the benefit of 3rd party front ends, perhaps it's best to not use the word "threshold" if we can get by with just the existing nomenclature.